site/src/pages/OrganizationSettingsPage/OrganizationMembersPageView.tsx

```
@@ -152,11 +152,7 @@ export const OrganizationMembersPageView: FC<
 												try {
 													await updateMemberRoles(member, roles);
 													displaySuccess("Roles updated successfully.");
-												} catch (error) {
-													displayError(
-														getErrorMessage(error, "Failed to update roles."),
-													);
-												}
+												} catch {}
```

Since we’re not catching anything, we can remove the try/catch block entirely to avoid hiding errors.

---

Yeah, it wouldn't have any bearing on the actual UI, but it feels like if something breaks, it'd be more valuable to have something still get logged to the console instead of having the error get swallowed entirely

Plus, a try/catch with an empty catch block and no comment explaining what we're doing feels weird